qir/qsharp/TargetPackages/QSharpVersion/NuGet.config

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+
+<!--
+    This configuration adds the Quantum Development Kit prerelease feed to the NuGet feeds.
+    This allows to reference prerelease packages from other parts of the Quantum Development Kit for testing purposes.
+
+    See https://docs.microsoft.com/en-us/nuget/consume-packages/configuring-nuget-behavior#how-settings-are-applied
+    for more information about how NuGet and the .NET Core SDK find package sources specified in NuGet.Config files.
+-->
+<configuration>
+  <packageSources>
+    <add key="local-packages" value="/Users/tfr/Downloads/" />

```

Comment:
@bettinaheim we need something non-local to go here. Do you have any suggestions?

Reply:
Soon. I'll push the packages to alpha as soon as I have the full e2e working.

Reply:
0.24.41900-alpha is pushed to alpha but has one issue that will interfere with Measure, 0.24.41925-alpha should fix that as well and hopefully will be pushed soon.